### ISSUE TITLE
Request: BFSAR Editing Support

### ISSUE THREAD

**Issue by nadrino** [KillzXGaming/Switch-Toolbox#257]
As mentionned in the title, it would be great if the Switch-Toolbox offered the possibility to edit BFSAR since there is absolutly no tool for doing this job.

BFSAR files contains audio data (maybe other types as well), and are often used for storing voiceover. **BFSAR support in Switch-Toolbox would be extremly useful for building translations mods for example.**

Currently there is only few rudimentary tools which are able to split those ([here for example](https://github.com/switch-stuff/BFSAR_Split)), but nothing to edit them. There is this guide tho, which explains how to replace one audio file manually, but it's pretty hard to follow and it is for the WiiU: [[Tutorial] How to replace the starman music in Mario Kart 8](https://gbatemp.net/threads/tutorial-how-to-replace-the-starman-music-in-mario-kart-8.478691/).

However, it seems like this format is pretty well understood ([Custom Mario Kart Wiki](http://mk8.tockdom.com/wiki/BFSAR_(File_Format))), so it might be possible to build a complete support for editing those files.

Finally I've seen Switch-Toolbox was intended to support those files years ago: ["There's many more features planned and coming soon in later versions! (BFSAR edits, Bfres animation edits, etc)"](https://github.com/KillzXGaming/Switch-Toolbox/releases). It seemd there was no news since then, so I'm sending this ticket to mention this feature request! 😄

**Comment by KillzXGaming** [comment]
BFSAR ending up being too audio heavy for me to mess with. It uses various audio files and sections combined into one fairly complex format. I don't really mess with audio much so it's not particularly a format I want to mess with so it's not planned anymore. Citric composer does have BFSAR support (but saving/editing make not work or is wip atm) I suggest checking that out. https://gota7.github.io/Citric-Composer. Also check their discord for more updates and newer builds https://discord.gg/gWheXh.
